Assets/01_Scripts/Spikes.cs

```diff
@@ -6,6 +6,7 @@ public class Spikes : MonoBehaviour
 {
     void OnCollisionEnter2D(Collision2D other)
     {
-        Destroy(other.gameObject);
+        
+        gameObject.SetActive(false);
```

Comment:
Shouldn't this be `other.gameObject.SetActive(false);` ? Otherwise it's going to applied to the object with the script.